astronomer/providers/apache/livy/hooks/livy.py

```diff
@@ -0,0 +1,426 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Same here, this license shouldn't be here!

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/178/commits/644dc92159393e3dfff0f4eb86da81430e6f6428)